build.gradle

```diff
@@ -90,7 +93,7 @@ ext.libraries = [
         presto_jdbc          : "com.facebook.presto:presto-jdbc:${versions.presto}",
         reflections          : "org.reflections:reflections:${versions.reflections}",
         bytebuddy            : "net.bytebuddy:byte-buddy:${versions.bytebuddy}",
-        jsch                 : "com.jcraft:jsch:${versions.jsch}",
+        jsch                 : "com.github.mwiede:jsch:${versions.jsch}",
```

Comment:
The old jsch was left unmaintained. This fork is maintained and has recent updates